lte/gateway/c/sctpd/src/sctpd.cpp

```diff
@@ -75,11 +80,27 @@ int signalHandler(int* end, std::unique_ptr<Server>& server,
   return 0;
 }
 
+static magma::mconfig::SctpD load_mconfig() {
+  magma::mconfig::SctpD mconfig;
+  if (!magma::load_service_mconfig_from_file(SCTPD_SERVICE, &mconfig)) {
+    mconfig.set_log_level(magma::orc8r::LogLevel::INFO);
+  }
+  return mconfig;
+}
+
 int main() {
   signalMask();
 
-  magma::init_logging("sctpd");
-  magma::set_verbosity(MDEBUG);
```

Comment:
We used INFO because all other services seem to use INFO as default in the mconfig. Was there a reason to set DEBUG for this service?

Reply:
not really. INFO level is good.